### PR TITLE
Add Keras unsupported argument list doc

### PR DESCRIPTION
**Pull request by hkvision**
## What changes were proposed in this pull request?

unsupported arguments & issues

## How was this patch tested?

Jenkins

### CODE REVIEW

File: pyspark/bigdl/keras/issues.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* For [`Embedding`](https://faroit.github.io/keras-docs/1.2.2/layers/embeddings/#embedding), `mask_zero` and `dropout` are not supported.
+* For [`PReLU`](https://faroit.github.io/keras-docs/1.2.2/layers/advanced-activations/#prelu), `init`, `weights` and `shared_axes` are not supported.
+* For [`ParametricSoftplus`](https://faroit.github.io/keras-docs/1.2.2/layers/advanced-activations/#parametricsoftplus), `weights` and `shared_axes` are not supported. Only `alpha_init = 1/beta_init` is supported.
+* For [`BatchNormalization`](https://faroit.github.io/keras-docs/1.2.2/layers/normalization/#batchnormalization), only `mode=0` is supported. Only `channel_first` (i.e. `dim_ordering='th'` with `axis=1`) and `channel_last` (i.e. `dim_ordering='tf'` with `axis=-1`) is supported. `gamma_regularizer` and `beta_regularizer` are not supported.
+* For [`Bidirectional`](https://faroit.github.io/keras-docs/1.2.2/layers/wrappers/#bidirectional) wrapper, only `return_sequences=True` is supported for RNNs. The arguments not supported by RNNs are not supported as well.
```
Comment (zhichao-li) [inline pyspark/bigdl/keras/issues.md]: Move item `Bidiractional` to just below `RNN`

File: pyspark/bigdl/keras/issues.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* For [`Embedding`](https://faroit.github.io/keras-docs/1.2.2/layers/embeddings/#embedding), `mask_zero` and `dropout` are not supported.
+* For [`PReLU`](https://faroit.github.io/keras-docs/1.2.2/layers/advanced-activations/#prelu), `init`, `weights` and `shared_axes` are not supported.
+* For [`ParametricSoftplus`](https://faroit.github.io/keras-docs/1.2.2/layers/advanced-activations/#parametricsoftplus), `weights` and `shared_axes` are not supported. Only `alpha_init = 1/beta_init` is supported.
+* For [`BatchNormalization`](https://faroit.github.io/keras-docs/1.2.2/layers/normalization/#batchnormalization), only `mode=0` is supported. Only `channel_first` (i.e. `dim_ordering='th'` with `axis=1`) and `channel_last` (i.e. `dim_ordering='tf'` with `axis=-1`) is supported. `gamma_regularizer` and `beta_regularizer` are not supported.
```
Comment (zhichao-li) [inline pyspark/bigdl/keras/issues.md]: remove `i.e`?

Comment (hkvision) [reply pyspark/bigdl/keras/issues.md]: ...

File: pyspark/bigdl/keras/issues.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+## Summary
+Up to now, we have generally supported __ALL__ the layers of [__Keras 1.2.2__](https://faroit.github.io/keras-docs/1.2.2/) to be loaded into BigDL.
+
+Self-defined Keras layers or [`Lambda`](https://faroit.github.io/keras-docs/1.2.2/layers/core/#lambda) layers are not supported for now. Weight sharing is not supported for now.
+
+This pages lists the so-far unsupported arguments for specific layers and some known issues.
+
+## Unsupported Layer Arguments
+For the following arguments, only the default values are expected and supported.
```
Comment (jason-dai) [inline pyspark/bigdl/keras/issues.md]: currently only the default values are expected and supported

File: pyspark/bigdl/keras/issues.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+## Summary
```
Comment (yiheng) [inline pyspark/bigdl/keras/issues.md]: Please move this page to the document site.

Comment (hkvision) [reply pyspark/bigdl/keras/issues.md]: Moved to `docs/docs/APIGuide/keras-issues.md` and added a link from `keras-support.md`

File: pyspark/bigdl/keras/issues.md (hunk 3 above)
Comment (hkvision) [reply pyspark/bigdl/keras/issues.md]: Updated.

File: pyspark/bigdl/keras/issues.md (hunk 1 above)
Comment (hkvision) [reply pyspark/bigdl/keras/issues.md]: Updated.

**Comment by hkvision** [comment]
@zhichao-li Rebased.

**Comment by zhichao-li** [comment]
LGTM

**Comment by hkvision** [comment]
jenkins passed http://172.168.2.101:8080/view/PR-V/job/BigDL-PR-V/637/

**Pull request merged**